cosmo_tester/framework/test_hosts.py

```diff
@@ -1115,9 +1115,13 @@ def _upload_blueprints_to_infrastructure_manager(self):
         self._logger.info(
             'Uploading test blueprints to infrastructure manager.'
         )
-        suffix = '-multi-net' if self.multi_net \
-            else '-ipv6-net' if self.ipv6_net \
-            else ''
+        suffix = ''
+        if self.ipv6_net:
```

Comment:
Let's make them -ipv6-net and -multi-net? That way if we do an ipv6 multi-net later it'll just work (if we name blueprints correctly).